client/app/account/edit/component/index.ts

```diff
@@ -0,0 +1 @@
+export { EditAccountForm } from './EditAccountForm'
```

Comment:
Hmm maybe we need to discuss this, but I personally prefer to put all components one layer under the feature directory, So this would be `/app/account/components`

Reply:
Fixed it. 
b23466c8ecaed82d91aa9509eecd76ee14937d34